mobile/src/bundles/common/components/overlay/overlay.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+import {
+    ActivityIndicator,
+    View,
+} from '~/bundles/common/components/components';
+
+import { Color } from '../../enums/enums';
+import { globalStyles } from '../../styles/global-styles';
+import { styles } from './styles';
+
+type OverlayProperties = {
+    isActive: boolean;
+    color?: string;
+};
+const Overlay: React.FC<OverlayProperties> = ({
+    isActive,
+    color = Color.PRIMARY,
+}) => {
+    if (!isActive) {
+        return null;
+    }
+    return (
+        <View
+            style={[
+                globalStyles.width100,
+                globalStyles.height100,
```

Comment:
```suggestion
                globalStyles.flex1,
```

Reply:
Flex: 1 didnt resize absolute component, or i should use it in another way?